src/utils/sort.ts

```diff
@@ -27,16 +26,16 @@ export function sortByDomain(data, property, direction = 'asc', domain) {
 
 export function sortByTime(data, property, direction = 'asc') {
   return data.sort((a, b) => {
-    const aDate = moment(a[property]);
-    const bDate = moment(b[property]);
+    const aDate = a[property].getTime();
+    const bDate = b[property].getTime();
```

Comment:
Is this a breaking change, as the data sets you pass in can no longer use string representations of dates?

The examples at https://swimlane.github.io/ngx-charts/ use strings and I'm thinking this will no longer work unless you map over your data sets and replace date-containing `string`s with `Date` instances before passing them to your chart.

``` typescript
{
    "name": "Mauritania",
    "series": [
      {
        "value": 5361,
        "name": "2016-09-24T03:54:47.499Z" // <-- cannot call `.getTime() on this
      },
      {
        "value": 4108,
        "name": "2016-09-15T11:10:41.923Z"
      }
    ]
}
```